Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM alpine
+FROM golang:1.17-alpine3.15 as builder
 
-RUN apk add ca-certificates --no-cache
+RUN apk add ca-certificates --no-cache make && update-ca-certificates
```

Comment:
We need to copy the car certificates into the final container now

Reply:
Done